src/utils/CortexLookup.js

```diff
@@ -80,6 +80,16 @@ const navigationFormZoomArray = [
   'offers:facets:element:facetselector:chosen:description',
   'offers:facets:element:facetselector:chosen:selector',
   'offers:facets:element:facetselector:chosen:selectaction',
+  'offers:sortattributes',
+  'offers:sortattributes:choice',
+  'offers:sortattributes:choice:description',
+  'offers:sortattributes:choice:selectaction',
+  'offers:sortattributes:choice:selector',
+  'offers:sortattributes:chosen',
+  'offers:sortattributes:chosen:description',
+  'offers:sortattributes:chosen:selectaction',
+  'offers:sortattributes:chosen:selector',
+  'offers:sortattributes:offersearchresult',
   'featuredoffers',
```

Comment:
Need to also add the following so `sortattributes` are returned when navigating from facet selection:

```
'sortattributes',
  'sortattributes:choice',
  'sortattributes:choice:description',
  'sortattributes:choice:selectaction',
  'sortattributes:choice:selector',
  'sortattributes:chosen',
  'sortattributes:chosen:description',
  'sortattributes:chosen:selectaction',
  'sortattributes:chosen:selector',
  'sortattributes:offersearchresult',
```